data/state/baseAccount.go

```diff
@@ -37,6 +38,11 @@ func (ba *baseAccount) DataTrieTracker() DataTrieTracker {
 	return ba.dataTrieTracker
 }
 
+// AccountDataHandler return the account data handler
+func (ba *baseAccount) AccountDataHandler() vmcommon.AccountDataHandler {
```

Comment:
Interesting pattern to avoid cyclic imports. Make 2 functions return the same thing but the function definition is different as to comply with a local interface :thinking:

Reply:
at least golang let's you do this.

Reply:
returns*